pkg/relay/ethereum/config.go

```diff
@@ -42,10 +40,20 @@ func (c ChainConfig) Validate() error {
 	if c.MaxRetryForInclusion == 0 {
 		errs = append(errs, fmt.Errorf("config attribute \"max_retry_for_inclusion\" is zero"))
 	}
+	if err := c.Signer.GetCachedValue().(SignerConfig).Validate(); err != nil {
```

Comment:
First you should check if `c.Signer` is non-nil.

Reply:
@bluele Thank you! I fixed it now.